src/Language/Haskell/Liquid/Transforms/CoreToLogic.hs

```diff
@@ -624,7 +624,8 @@ instance Simplify C.CoreExpr where
   simplify allowTC (C.Let xes e)
     = C.Let (simplify allowTC xes) (simplify allowTC e)
   simplify allowTC (C.Case e x _t alts@[(_,_,ee),_,_]) | isBangInteger alts
-    = Misc.traceShow ("To simplify allowTC case") $ 
+  -- XXX(matt): seems to be for debugging?
```

Comment:
Requesting confirmation this comment is accurate.

Reply:
yes, tracing is for debugging